### DISCUSSION TITLE
Announcing the Hasura Super App! 🚀

### DISCUSSION THREAD

**Discussion by mehtavishwa30** [hasura/graphql-engine#7281] [General] [answered]
We get asked a lot whether our users can build modern apps with complex business logic using Hasura. And we found this question to be interesting. It made us think how we could help our users understand the core Hasura features better and learn how to empower themselves with said features! 🤔

So we built a reference app to showcase an awesome lineup of Hasura features that enable you to build complex modern apps and move to the market faster with them!

📢  Announcing Hasura Super App—A fully-featured, free, & open source reference app that showcases how to leverage the Hasura superpowers to build scalable, secure, and high-performance business applications.

🚀 🎉 🚀 🎉 🚀 🎉 🚀 🎉 

Check it out and do let us know what you think!
➡️ https://hasura.io/reference-app/

**Comment by Adron** [comment, marked as answer]
💥👍🏻
